package.json

```
@@ -12,7 +12,8 @@
     "dev:gatsby": "gatsby develop -H 0.0.0.0",
     "dev:prettier": "onchange \"@(*.css|*.js|*.json|*.ts|@(src|types)/**/*.@(ts|tsx))\" -- prettier --write {{changed}}",
     "codegen": "graphql-codegen --config codegen.yml",
-    "dev": "run-p dev:**"
+    "dev": "run-p dev:**",
+    "tsc": "tsc --noemit"
```

```suggestion
    "tsc": "tsc --noEmit"
```

It also works but `--noEmit` seems right representation by following [compiler-options document][typescript-compiler-options-docs].

[typescript-compiler-options-docs]: https://www.typescriptlang.org/docs/handbook/compiler-options.html